scripts/app.ts

```diff
@@ -70,7 +71,9 @@ function removeLinks(workItem: TFS_Wit_Contracts.WorkItem, linkedWorkItemIds: nu
 
 function addRelations(workItem: TFS_Wit_Contracts.WorkItem, relations: TFS_Wit_Contracts.WorkItemRelation[]): IPromise<TFS_Wit_Contracts.WorkItem> {
```

Comment:
Thanks for switching from Q to native promises, for consistency should we also change IPromise to Promise?

Reply:
When I updated dependencies, Q was causing a mess. I think it was a conflict between what versions of Q vss-web-extension-sdk and react are using. Seems like IPromise as the return type allows compatibility with both return types used in that function. I'm not 100% familiar with the intricacies of the various types of promises so if there is a better approach for better consistency I'm curious exactly what it'd look like.